tso_session/intro_TSO_data_pipeline.ipynb

```diff
@@ -136,11 +136,74 @@
     "First the imports.\n"
```

Comment:
<p>You can ignore the first two warnings here; <code>flake8</code> isn't recognizing that the <code>%set_env</code> statements are not Python code.</p><p> </p><p>You can resolve the other warnings by either separating environment variable assignment into its own cell or by moving it after the imports in this one.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/jwebbinar_prep/pull/110/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>done</p>